EarlGrey/Additions/__NSCFLocalDataTask_GREYAdditions.m

```diff
@@ -64,7 +64,7 @@ - (void)greyswizzled_resume {
     objc_setAssociatedObject(self,
                              @selector(greyswizzled_setState:),
                              elementID,
-                             OBJC_ASSOCIATION_RETAIN_NONATOMIC);
+                             OBJC_ASSOCIATION_RETAIN);
```

Comment:
It might be helpful to add a comment (something [like this](http://stackoverflow.com/a/29935110) to explain why retain is necessary to avoid the race condition.